packages/snaps-execution-environments/src/common/utils.ts

```diff
@@ -122,12 +115,6 @@ export function assertSnapOutboundRequest(args: RequestArguments) {
       },
     }),
   );
-  assertStruct(
```

Comment:
As part of input sanitation we already validate that requests from Snaps are valid JSON, making this unnecessary